balancer_wrapper.go

```diff
@@ -413,6 +420,20 @@ func (acbw *acBalancerWrapper) closeProducers() {
 	}
 }
 
+// HealthCheckOptions are the options to configure the health check producer.
+//
+// # Experimental
+//
+// Notice: This type is EXPERIMENTAL and may be changed or removed in a
+// later release.
+type HealthCheckOptions struct {
+	// ServiceName of the gRPC service running on the server for reporting health state.
```

Comment:
Nit: Wrap comment at 80-cols?

Reply:
Removed the exported struct as discussed above.